### DISCUSSION TITLE
Non-Github version of pybricks documentation

### DISCUSSION THREAD

**Discussion by axfor8** [pybricks/support#311] [Q&A] [answered]
Firstly, I'm a 100% Github novice, so I apologise that this question is very basic. I have searched and could not find an answer.

I am looking for a way to access the pybricks documentation (like that found at [https://pybricks.com/ev3-micropython/](url)). I have a class who are working with pybricks that would benefit from accessing the docs, however they are unable to use the link because Github is blocked at a state-wide level for students.

Any suggestions or pointers to a PDF or other URL with the same docs would be greatly appreciated.

**Comment by XilefTech** [comment]
Have you tried using opera with it's free "VPN" to bypass the restriction? for me that worked well so far to bypass any restrictions/website blocking in my school. Also I've found [this pdf doc](https://le-www-live-s.legocdn.com/sc/media/files/ev3-micropython/ev3micropythonv100-71d3f28c59a1e766e92a59ff8500818e.pdf) for pybricks 1.0. There sure are a some changes in V2 for EV3 yet it should be at least a start for simple functions like for example motors & stuff.

**Reply by axfor8** [reply]
Thanks for the suggestion, but I don't want to go down the rabbit hole of bypassing restrictions.

**Comment by laurensvalk** [comment, marked as answer]
Thanks for this question. We can certainly publish the PDF of version 2.0 (corresponding to the EV3 education version). I should be able to do this tomorrow or Monday.

**Reply by axfor8** [reply]
That would be absolutely fantastic! Thank you!

**Reply by laurensvalk** [reply]
Is this what you are looking for?

[ev3-micropython-v2.0.0.pdf](https://github.com/pybricks/support/files/6342384/ev3-micropython-v2.0.0.pdf)

We might move this to the main doc page to make it easier to find for others as well.

**Reply by axfor8** [reply]
Yes, that's exactly what I was hoping for. Thank you so much!

**Comment by TheGreyDiamond** [comment]
The real question is why is Github.com blocked for students?

**Reply by axfor8** [reply]
I'm sure there would be a number of reasons, but network filtering is controlled at a state level and it's not something I can change at the local level.

**Reply by TheGreyDiamond** [reply]
Yeah, that's kinda understandable.
